### ISSUE TITLE
Memcache cannot pickle a generator

### ISSUE THREAD

**Issue by swdream** [bogdal/sentry-youtrack#16]
When creating new youtrack issue:
```
sentry[2721] ERROR django.request base.handle_uncaught_exception:224 Internal Server Error: /team/project/issues/6052/actions/youtrack/
    Traceback (most recent call last):
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/core/handlers/base.py", line 112, in get_response
        response = wrapped_callback(request, *callback_args, **callback_kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/views/generic/base.py", line 69, in view
        return self.dispatch(request, *args, **kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 29, in _wrapper
        return bound_func(*args, **kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 99, in _wrapped_view
        response = view_func(request, *args, **kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/utils/decorators.py", line 25, in bound_func
        return func(self, *args2, **kwargs2)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/web/frontend/base.py", line 172, in dispatch
        return self.handle(request, *args, **kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/web/frontend/group_plugin_action.py", line 25, in handle
        response = plugin.get_view_response(request, group)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 113, in get_view_response
        return super(YouTrackPlugin, self).get_view_response(request, group)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/plugins/base/v1.py", line 296, in get_view_response
        response = self.view(request, group)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 131, in view
        return view(request, group, **kwargs)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/plugins/bases/issue.py", line 169, in view
        form = self.get_new_issue_form(request, group, event)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 77, in get_new_issue_form
        project_fields=self.get_project_fields(group.project),
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/plugin.py", line 57, in get_project_fields
        return cached_fields(self.get_option('ignore_fields', project))
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../sentry_youtrack/utils.py", line 16, in wrapper
        cache.set(key, result, timeout)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/sentry/../django/core/cache/backends/memcached.py", line 82, in set
        self._cache.set(key, value, self._get_memcache_timeout(timeout))
      File "/usr/local/sentry/local/lib/python2.7/site-packages/memcache.py", line 740, in set
        return self._set("set", key, val, time, min_compress_len, noreply)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/memcache.py", line 1060, in _set
        return _unsafe_set()
      File "/usr/local/sentry/local/lib/python2.7/site-packages/memcache.py", line 1034, in _unsafe_set
        store_info = self._val_to_store_info(val, min_compress_len)
      File "/usr/local/sentry/local/lib/python2.7/site-packages/memcache.py", line 998, in _val_to_store_info
        pickler.dump(val)
    PicklingError: Can't pickle <type 'generator'>: attribute lookup __builtin__.generator failed
```
sentry version: 8.4.0
sentry-youtrack plugin version: 0.3.1
YouTrack version: youtrack-6.5.17105
python-memcached version: 1.53 (I tried to upgrade python-memcached to 1.58 but still suffer this issue)

Thanks & best regards.

**Comment by bogdal** [comment]
@swdream Fixed in `0.3.2` version.

**Comment by swdream** [comment]
@bogdal thanks a lot
